### DISCUSSION TITLE
Video stream continuous face recognition.

### DISCUSSION THREAD

**Discussion by trashcluster** [exadel-inc/CompreFace#403] [Q&A]
Is there a way to enable continuous face recognition from a video stream ?

**Comment by pospielov** [comment]
CompreFace doesn't support video itself. 
You can read the video, split it into frames and then send them to CompreFace.
Probably we will add this in SDK when we create one :)

**Reply by uma-sathish** [reply]
Now you will add the feature to the SDK

**Reply by pospielov** [reply]
Sorry, SDK doesn't support video. 
But we have a demo of how to use a web camera video stream to recognize faces, e.g.:
https://github.com/exadel-inc/compreface-python-sdk/blob/main/webcam_demo/compreface_webcam_recognition_demo.py
